### PR TITLE
Identity area tj

### LINKED ISSUE #4: Create CardArea
As a user I want to see all my ID cards in one area

### LINKED ISSUE #5: Connect cardArea to redux
As a dev I want my cardArea to be connected to the ID part of state

### LINKED ISSUE #9: ID page component
As a user I want to be able to see a backstory for each of my ID's in a separate area as well as the page header

### LINKED ISSUE #10: ID click event
As a user when I click on an ID I want the backstory for that ID to appear in a separate area.

### PR DESCRIPTION
**Pull request by taylorjohnson141**
### What does this PR do?
This PR adds the functionality to view the identity area and for each identify to generate a backstory and mission when clicked

### Where should the reviewer start?

IDpage.js

### How should this be manually tested?
npm start and run through folders

### Any background context you want to provide?
Needs testing and styling

### What are the relevant tickets?
closes #4 
closes #5 
closes #9 
closes #10

### CODE REVIEW

File: src/components/StoryArea/StoryArea.js [hunk 1]
```diff
@@ -1,10 +1,21 @@
...
+      ${currentIdentity['last-name']}. 
+      You work for ${currentIdentity['company-name']} as a/an ${currentIdentity['occupation-name']}. 
+      You have ${currentIdentity.numbers1} children and your spouse’s name is ${
+        currentIdentity['first-name2']}. The two of you met at ${currentIdentity['restaurant-name']}. 
+      You have a ${currentIdentity['singular-animal-name']} and two ${currentIdentity['plural-animal-name']}  
+      In your free time, you like to coach a youth 
+      ${currentIdentity['sport-name']} team and play ${currentIdentity['game-name']} on weekends with your social club. 
+      You enjoy ${currentIdentity.genre1} novels and watching ${currentIdentity.genre2} movies. 
+      You have ${currentIdentity.numbers2} close friends who often call you by your nickname, 
+      the ${currentIdentity['color-name']} ${currentIdentity['zoo-animal']}`}.
```
Comment (ericberglund117) [inline src/components/StoryArea/StoryArea.js:18]: Thanks for writing this out! It looks good.

File: src/components/IDPage/IDPage.js [hunk 2]
```diff
@@ -1,15 +1,26 @@
...
-const IDPage = () => {
   return (
     <section>
-      <h2>Name's Identity</h2>
       <h3>Your Backstory</h3>
-      <StoryArea />
+      <StoryArea currentIdentity = {currentIdentity}  currentCountry ={currentCountry}/>
       <h3>Your Mission</h3>
-      <MissionArea />
+      <MissionArea currentCountry ={currentCountry}/>
```
Comment (ericberglund117) [inline src/components/IDPage/IDPage.js:23]: I'm interested in the other way using state management and Redux, but this looks good so far. I'm glad you were able to use the data for the mission and story

File: src/components/Identity/Identity.js [hunk 3]
```diff
@@ -1,18 +1,19 @@
...
-      <h3>First Name</h3>
-      <h2>xxx</h2>
-      <h3>Date of Birth</h3>
-      <h2>xxx</h2>
-      <h3>Place of Birth</h3>
-      <h2>xxx</h2>
-    </section>
+    <Link to = {`/${String(identities.id)}`}>
+      <section >
+        <img alt = 'spy-pic' />
```
Comment (ericberglund117) [inline src/components/Identity/Identity.js:10]: Were we going to use a generic stock photo for this spy-pic?

**Review by ericberglund117 — APPROVED** [review]

File: src/components/Identity/Identity.js (hunk 3 above)
Comment (timkeresey) [reply src/components/Identity/Identity.js:10]: I was thinking like a silhouette 'anonymous person' photo

File: src/components/IDPage/IDPage.js (hunk 2 above)
Comment (timkeresey) [reply src/components/IDPage/IDPage.js:23]: Yes, this works fine and implements react well. I think that knowing what the alternative is with redux principles is good, and either implementation is fine.

**Pull request merged**